drupal/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Ignore directories generated by Composer
+/.editorconfig
+/.gitattributes
+/vendor/
+/web/_ping.php
+/web/.gitignore
+/web/core/
+/web/modules/.gitignore
+/web/modules/contrib/
+/web/themes/.gitignore
+/web/themes/contrib/
+/web/profiles/.gitignore
+/web/profiles/contrib/
+/web/libraries/
+/web/sites/.gitignore
+/web/sites/default/.gitignore
+
+/.idea
```

Comment:
.idea is already ignored in root .gitignore

Reply:
done in https://github.com/wunderio/next4drupal-project/pull/10/commits/9b36fbdd4fbb64fc73d22620eca857d4abd75adc